ksql-engine/src/main/java/io/confluent/ksql/planner/LogicalPlanner.java

```diff
@@ -211,7 +211,7 @@ private ProjectNode buildProjectNode(final PlanNode sourcePlanNode) {
     return new ProjectNode(
         new PlanNodeId("Project"),
         sourcePlanNode,
-        projectionSchema,
+        projectionSchema.build(),
```

Comment:
fix a nit: this ensures a immutable schema is passed, rather than a mutable one.